packages/disploy/src/structs/Guild.ts

```diff
@@ -90,9 +90,12 @@ export class Guild extends Base {
 	 * 	The NSFW level for this guild.
 	 */
 	public nsfwLevel!: number;
-
+	/**
+	 * The raw data of the guild.
+	 */
+	declare public raw: APIGuild;
 	public constructor(app: App, raw: APIGuild) {
-		super(app);
+		super(app, raw);
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.